autoload/iced/nrepl/document.vim

```diff
@@ -128,23 +128,36 @@ function! iced#nrepl#document#open(symbol) abort
 endfunction
 
 function! s:one_line_doc(resp) abort
+  let resp = a:resp
+  if type(a:resp) == v:t_list
+    if len(a:resp) > 0
+      let resp = a:resp[0]
+    else
+      return
+    endif
+  endif
```

Comment:
It seems better to modify autoload/iced/nrepl/op/cider.vim, because `info` op is used by everywhere.
We need operation handler for `info` op like follows:
```
function! s:info_handler(resp, _) abort
  if type(a:resp) == v:t_list
    " ...
  else
    return a:resp
  endif
endfunction
```
and add `call iced#nrepl#register_handler('info', funcref('s:info_handler'))` to the bottom of the file.